mlflow/server/js/src/experiment-tracking/components/CompareRunBox.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Row, Col, Select } from 'antd';
```

Comment:
Can we use `databricks/design-system` here instead of antd similar to this: https://github.com/mlflow/mlflow/blob/3fbab3bcd04ba29277ea6cfa96af37ec94392f45/mlflow/server/js/src/experiment-tracking/components/ExperimentView.js#L15

Reply:
`databricks/design-system` doesn't currently have support for `OptGroup`, so you'll have to do something like this
```js
import { Select } from '@databricks/design-system';
import { Select as AntSelect } from 'antd';

const { Option } = Select
const { OptGroup } = AntSelect
```

Reply:
With the code @xanderwebs suggested, the param/metric selectors look like this:

<img width="484" alt="image" src="https://user-images.githubusercontent.com/17039389/181137856-ed30c30c-b248-4119-b4d6-a09056760900.png">

Reply:
Should we use antd's `Option` and `OptGroup` as the scatter and contour plots use it?

Reply:
Discussed with @hubertzub-db on this yesterday. We recently migrated the contour and scatter plot to `@databricks/design-system` (with `antd.Select.OptGroup`). When we export that change in the next UI sync, we can also update the box plot.